google/ggrc-core
# GGRC-5549 TreeView is not refreshed on object page while adding a cycle task

# Issue description

TreeView is not refreshed on object page while adding a cycle task.

# Steps to test the changes

Steps to reproduce:
1. Open any Control page
2. Add tab > Workflow task
3. Create a cycle task

*Actual Result:* Cycle task is not displayed in tree view after creating. User has to refresh the page
*Expected Result:* TreeView should be refreshed and cycle task should be displayed after creating

# Solution description

I described needed details in comments.

# Sanity checklist




# PR Review checklist
